docs/source/usage/workflows/particleFilters.rst

```diff
@@ -60,10 +60,108 @@ and add ``ParticlesForwardPinhole`` to the ``AllParticleFilters`` list:
 
 .. code:: cpp
 
-   using AllParticleFilters = MakeSeq_t<
-       All,
-       ParticlesForwardPinhole
-   >;
+       using AllParticleFilters = MakeSeq_t<
+           All,
+           ParticlesForwardPinhole
+       >;
+
+   } // namespace filter
+   } // namespace particles
+   } // namespace picongpu
+
+Filtering Based on Global Position
+""""""""""""""""""""""""""""""""""
+
+A particle only stores its relative position inside the cell.
+Thus, with a simple filter like one shown above, it is not possible to get global position of a particle.
+However, there are helper wrapper filters that provide such information in addition to the particle data.
+
+For a special case of slicing along one axis there is a simple existing filter that only needs to be parametrized:
+
+.. code:: cpp
+
+   namespace picongpu
+   {
+   namespace particles
+   {
+   namespace filter
+   {
+       namespace detail
+       {
+           //! Parameters to be used with RelativeGlobalDomainPosition, change the values inside
+           struct SliceParam
+           {
+               // Lower bound in relative coordinates: global domain is [0.0, 1.0]
+               static constexpr float_X lowerBound = 0.55_X;
+
+               // Upper bound in relative coordinates
+               static constexpr float_X upperBound = 0.6_X;
+
+               // Axis: x = 0; y= 1; z = 2
+               static constexpr uint32_t dimension = 0;
+
+               // Text name of the filter, will be used in .cfg file
+               static constexpr char const* name = "slice";
+           };
+
+           //! Use the existing RelativeGlobalDomainPosition filter with our parameters
+           using Slice = RelativeGlobalDomainPosition<SliceParam>;
+       }
+
+and add ``detail::Slice`` to the ``AllParticleFilters`` list:
+
+.. code:: cpp
+
+       using AllParticleFilters = MakeSeq_t<
+           All,
+           detail::Slice
+       >;
+
+   } // namespace filter
+   } // namespace particles
+   } // namespace picongpu
+
+For a more general case of filtering based on cell index (possibly combined with other particle properties) use the following pattern:
+
+.. code:: cpp
+
+   namespace picongpu
+   {
+   namespace particles
+   {
+   namespace filter
+   {
+       namespace detail
+       {
+           struct AreaFilter
+           {
+               static constexpr char const* name = "areaFilter";
+
+               template<typename T_Particle>
+               HDINLINE bool operator()(
+                   DataSpace<simDim> const totalCellOffset,
+                   T_Particle const & particle
+               )
+               {
+                   /* Here totalCellOffset is cell index of the particle in the total coordinate system.
```

Comment:
```suggestion
                   /* Here totalCellOffset is cell index of the particle in the total coordinate system.
```
```suggestion
                   /* Here totalCellOffset is the cell index of the particle in the total coordinate system.
```

Reply:
I do not know, why the diff appears twice (and once without any changes - seems to be an issue with GitHub - I deleted the comment, but the same issue appeared again.)